src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesLauncher.java

```diff
@@ -68,7 +67,7 @@ public class KubernetesLauncher extends JNLPLauncher {
 
     private static final Logger LOGGER = Logger.getLogger(KubernetesLauncher.class.getName());
 
-    private final AtomicBoolean launched = new AtomicBoolean(false);
+    private volatile boolean launched = false;
```

Comment:
FTR: this is not directly related or needed for the goal of this PR, but it was requested by @jglick in a review.

Reply:
(but consider https://github.com/jenkinsci/kubernetes-plugin/pull/1561#discussion_r1617565856)